themes/default/content/docs/pulumi-cloud/access-management/access-tokens.md

```diff
@@ -108,6 +108,14 @@ Navigate to your organization and then:
 
 The token must have a name that is unique among all organization and team access tokens in the organization., including deleted tokens. This allows tokens taking operations on behalf of your organization to be identifiable in the event that one is compromised. Any other organization admin can delete this token; it is not owned by the admin which created it. Creation of organization access tokens is logged as an audit log event.
 
+#### Admin organization access tokens
+
+Admin organization access tokens (admin tokens) are a specialized type of token that grants full administrator permissions within an organization. They provide unrestricted access to perform administrative operations on Pulumi resources throughout the entire organization. Namelly, these can be used to manage Policy-as-Code, org membership, and export audit logs. Admin tokens cannot be used to generate or delete additional organization tokens.
```

Comment:
>Namelly, these can be used to manage Policy-as-Code, org membership, and export audit logs.

As written, this sounds like these are the _only_ things they can do, but I'm assuming that isn't the case, and that admin tokens allow you to do anything a regular org admin can do except create or delete org tokens. Is that right?

Reply:
Good question. I couldn't find a way to word this that I liked yesterday. You have this correct. The intention was to provide a few examples of features this enables without requiring that the user navigate to another page to check what privileges administrators have.

Org tokens have the ability to generate team tokens. They cannot generate personal access tokens because this is not applicable (they aren't a real user account). They are prohibited from managing other organization tokens as a requirements choice/safeguard, though it's possible we might drop this restriction at some point.

Any suggestions to better communicate this?

Reply:
Ok cool, thanks for confirming. Would this maybe help?

```suggestion
Admin organization access tokens (or admin tokens) are organization tokens with elevated, administrator-level privileges. Admin tokens allow automated processes to perform any operation supported for organization administrators except for the creation or deletion of other organization tokens. 
```

This way, you can leave it to the docs to explain what's allowed and what isn't for administrators, and if y'all decide to remove the org-token restriction at some point, you can do that pretty easily. Feel free to accept or adjust -- just offering as something to consider if you like.

Reply:
I like this, sounds good!